src/main/java/it/gov/pagopa/cgn/portal/email/EmailNotificationFacade.java

```diff
@@ -224,11 +234,12 @@ public EmailNotificationFacade(TemplateEngine htmlTemplateEngine,
         this.configProperties = configProperties;
     }
 
-    private EmailParams createEmailParams(String mailTo, String subject, String body, String failureMessage) {
-        return createEmailParams(mailTo, Optional.empty(), subject, body, failureMessage);
+    private EmailParams createEmailParams(String mailTo, List<String> ccList, String subject, String body, String failureMessage) {
```

Comment:
this method was here to hide optional parameters, I would have kept it with only mandatory parameters...